x-pack/filebeat/input/azureblobstorage/input_test.go

```diff
@@ -473,6 +473,132 @@ func Test_StorageClient(t *testing.T) {
 				mock.Beatscontainer_2_blob_data3_json:  true,
 			},
 		},
+		{
+			name: "CustomContentTypeUnsupported",
+			baseConfig: map[string]interface{}{
+				"account_name":                        "beatsblobnew",
+				"auth.shared_credentials.account_key": "7pfLm1betGiRyyABEM/RFrLYlafLZHbLtGhB52LkWVeBxE7la9mIvk6YYAbQKYE/f0GdhiaOZeV8+AStsAdr/Q==",
+				"max_workers":                         2,
+				"poll":                                true,
+				"poll_interval":                       "10s",
+				"containers": []map[string]interface{}{
+					{
+						"name":                  beatsGzJSONContainer,
+						"content_type":          "application/xyz-plain",
+						"override_content_type": true,
+					},
+				},
+			},
+			mockHandler: mock.AzureStorageFileServer,
+			expected: map[string]bool{
+				"job with jobId beatsgzjsoncontainer-multiline.json.gz-worker-0 encountered an error: content-type application/xyz-plain not supported": true,
+			},
+		},
+		{
+			name: "CustomEncodingUnsupported",
```

Comment:
Is this unsupported or incorrect/mismatched?

Reply:
comment [here](https://github.com/elastic/beats/pull/44583#discussion_r2122854190)